packages/ui/src/documents/__tests__/Nfts.test.tsx

```diff
@@ -17,6 +15,7 @@ describe('Nfts', () => {
     {
       label: 'First Registry',
       address: '0xFirstRegistry',
+      asset_manager_address: '0xFirstRegistry',
```

Comment:
Maybe we should use valid eth addresses? Many functions should validate them.

Reply:
i can add this to the UI validation on the schema 👍